src/modules/blockchainApplication/manage/hooks/useCurrentApplication.js

```diff
@@ -0,0 +1,15 @@
+import { useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { selectCurrentApplication } from '../store/selectors';
+import { setCurrentApplication } from '../store/action';
+
+// eslint-disable-next-line
```

Comment:
Which is the reason eslint disabled here?

Reply:
we try to use normal export, (i remember two month ago we discuss about it) the reason is easy can export all hooks or utils, and because the import name is equal to the export name its much easier to find the function

Reply:
`import { useCurrentApplication } from `[smth]/useCurrentApplication.js` looks strange to me. I think `import useCurrentApplicatio from `[smth]/useCurrentApplication.js` is more correct as function name is same as file name. I would favour `export default useCurrentApplication` in this case. Avoid eslint expections is a good practise in my opinion

Reply:
we can talk about it in style guideline process please skip it, for now, we already implementation on this way

Reply:
I think the exact rull to disable can be used i.e
```suggestion
// eslint-disable-next-line import/prefer-default-export
```

Reply:
That function is that way because you in a PR review told me to do it that way.